Userland/Services/SpiceAgent/Message.h

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2023, Caoimhe Byrne <caoimhebyrne06@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Forward.h>
+#include <AK/Vector.h>
+
+namespace SpiceAgent {
+
+static constexpr u32 AGENT_PROTOCOL = 1;
+
+// Used to communicate what the client/or server is capable of
+// TODO(Caoimhe): Document these types
```

Comment:
The project doesn't use the TODO(person) style of TODO, simply TODO will suffice. 

Although, a link to the spice spec or wiki would probably be better :^)